Makefile

```diff
@@ -170,10 +170,10 @@ endif
 bundle: generate
 	operator-sdk generate kustomize manifests -q;\
 	VERSION=$$(cat ./version/version.go | grep -Po '(?<=Version =\s").*(?=")');\
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $${VERSION} $(BUNDLE_METADATA_OPTS);\
-	operator-sdk bundle validate ./bundle;\
+	$(KUSTOMIZE) build config/crd | operator-sdk generate bundle -q --overwrite --version $${VERSION} $(BUNDLE_METADATA_OPTS)
```

Comment:
I missed problem as kustomize was not configured properly, to be safe here - please add `make kustomize` before 173

Reply:
We don't need it anymore. I got rid of it and it comes from the ci image. We need to only use the docker image anytime we need to run something for safety